### PR TITLE
remove extra new line before error messages

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
The diff format is updated according to #1813

### CODE REVIEW

File: e2e/testdata/fn-render/success-stdout/.expected/diff.patch [hunk 1]
```diff
@@ -1,15 +1,14 @@
 diff --git a/resources.yaml b/resources.yaml
-index 7a494c9..c77c357 100644
+index 7a494c9..a9dd224 100644
```
Comment (mortent) [inline e2e/testdata/fn-render/success-stdout/.expected/diff.patch:2]: Why this change? There doesn't seem to be anything in the rest of the PR that would change the format of the output.

Comment (Shell32-Natsu) [reply e2e/testdata/fn-render/success-stdout/.expected/diff.patch:2]: I think the output format is changes in #1813

**Review by mortent — APPROVED** [review]
